### PR TITLE
Weather app

### PR DESCRIPTION
**Pull request by JohannRodriguez**
- User can input any city and it will be stored
- From the storage, a URL is created to fetch to the API site
- The temperature and weather are displayed to the user
- User can change the temperature system

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "weather-app",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (trekab) [inline package.json:8]: Add necessary run commands like `build`, `development`, e.t.c

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Hi @JohannRodriguez! Well done implementing the required functionality for this project. :clap: There are a few details for you to work on before having this project approved though. These include:
- [x] Add necessary run common commands like `build`, `development` should be in `package.json` inside `scripts`. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)). Check comment highlighted in file below.
- [x] Your `live demo` link in the `README.md` file doesn't work. Please update this with a deployed version of your app.
- [x] Your setup instructions in the `README.md` file are not descriptive enough. Add more details so that setting up a local copy of your project is more convenient.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes :recycle:
### Good job :+1: on implementing some of the required changes, however, the npm is still throwing an error when I try running `npm run start` or `npm start`

**_Fix this and submit for another review_**

#### _Reviewed by Temitayo Adeleke_

**Review by MarioDena — CHANGES_REQUESTED** [review]
Hello! Good work implementing the requested changes! 🥇 

However, the linter checks for your project are not passing.

Changes required 🔴 

- [ ] Be sure that the project pass all linter checks.

Cheers!

**Review by Dandush03 — APPROVED** [review]
# Status: :rocket: Project Approved :rocket: 

### Great Work! Keep It Up :muscle: 

# Happy Coding! :nerd_face: :computer:

**Pull request merged**